CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
+- Fix: add missed active attributes (with expressions) for ctxt for evaluate explicitAttrs expression

```

Comment:
```suggestion
- Fix: add missed active attributes (with expressions) for ctxt for evaluate explicitAttrs expression (#1351)
```

given this comment https://github.com/telefonicaid/iotagent-node-lib/issues/1351#issuecomment-1534203621